### ISSUE TITLE
False positive on ReusedModifierInstance with theme as direct child

### ISSUE THREAD

**Issue by mike-wideman** [appKODE/detekt-rules-compose#28]
Hello!

The following causes `ReusedModifierInstance` to fire:

```
@Composable
fun MyComposable(
    modifier: Modifier = Modifier
) {
    MaterialTheme {
        Row(
            modifier = modifier,
        ) { }
    }
}
```

This seems like a false positive given that the direct child is a theme.

**Comment by dimsuz** [comment]
Thank you for the report! 

This is a duplicate of #5 and is already fixed in `master`. 

The upcoming release (I plan to publish it in a few days) will contain the fix. For now you can temporarily `@Suppress` this false positive.

You'll have to use detekt in a type [resolution mode](https://detekt.dev/docs/gettingstarted/type-resolution/) though (which is preferable in any case, many useful rules will depend on it).
